demo/harassment-alert-create.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Signaler Harcèlement Demo</title>
+    <script type="module" src="/lib/geocity-wc-map.js"></script>
+  </head>
+  <body>
+    <div style="width: 100%; height: 80vh;margin-left: auto;margin-right: auto">
+        <openlayers-element  options='{ "enableDraw": false, 
+                                        "information": { "duration": 5000, "title": "Signaler un harcèlement", "content": "Positionner le centre de la cible à l’emplacement où le harcèlement a lieux" },
+                                        "enableGeolocation": true,
+                                        "enableCenterButton": true,
+                                        "enableRotation": true,
+                                        "mode": {
+                                            "type": "create"
+                                        },
+                                        "geolocationInformation": {
+                                            "displayBox": true,
+                                            "reverseLocation": false,
+                                            "currentLocation": true
+                                        },
+                                        "notifications": [
+                                        {
+                                            "type": "warning",
+                                            "message": "Veuillez zoomer davantage avant de pouvoir sélectionner un emplacement.",
+                                            "rule": {
+                                                "type": "ZOOM_CONSTRAINT",
+                                                "minZoom": 16
+                                            }
+                                        },
+                                        {
+                                            "type": "info",
+                                            "message": "Déplacez la carte pour que l’endroit désiré soit au centre de la cible",
+                                                "rule": {
+                                                "type": "MOVE_TARGET"
+                                            }
+                                        }
```

Comment:
does not seem consistent for "create" mode

Reply:
done